test/vcr_cassettes/PlaidDepositSwitchTest_test_all_routes.yml

```diff
@@ -0,0 +1,395 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: https://sandbox.plaid.com/item/import
```

Comment:
Is it expected for the test fixtures to contain `https://sandbox.plaid.com/item/import` in two places? The other place is https://github.com/plaid/plaid-ruby/pull/251/files#diff-95b4a2fae80e689eee848f1ac2eaff8eR5

Reply:
Oops, this looks like an error due to the VCR being ran twice, since the behavior of the VCR is to append to files and not replace them. will remove the second request. thanks for catching this!